### PR TITLE
Moves sources to their own column. fixes #21

### LINKED ISSUE #21: Sources Formatting Suggestion
Instead of having the sources be linked out on the company name, I believe it would make more sense to make another column and have Wikipedia-esque sources in the form of [1][2][3] etc.

It's inevitable that there will be several sources, especially with geo-specific WFH regulations.

### PR DESCRIPTION
**Pull request by trafnar**
Fixes #21 

- Moves sources to their own column
- Improves formatting of markdown source (done automatically with prettier.io)
- Changes "WFH" to "WFH Policy" (since this column is wide, there is room for the extra word)
- Changes "Last Updated" to "Updated" (to keep that column no wider than it needs to be)

If you hate the changes to markdown formatting, let me know.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,32 +4,32 @@ This is the running list of what in tech has been affected by COVID-19. Pull req
...
-| [Typless](https://typless.com/2020/03/05/switching-to-fully-remote-work/) | Required | Restricted | Restricted | Restricted | 2020-03-04 |
-| Yelp | Encouraged | Restricted | ? | ? | 2020-03-03 |
+| Company    | WFH Policy                                  | Travel     | Visitors   | Events     | Updated    | Source                                                                                                                                                | Po  |
+| ---------- | ------------------------------------------- | ---------- | ---------- | ---------- | ---------- | ----------------------------------------------------------------------------------------------------------------------------------------------------- | --- |
+| Amazon     | Encouraged, Recommended for Seattle to 3/31 | Restricted | Restricted | Restricted | 2020-03-04 | [businessinsider.com](https://www.businessinsider.com/companies-asking-employees-to-work-from-home-due-to-coronavirus-2020)                           |
+| Apple      | None                                        | Restricted | ?          | ?          | 2020-03-03 |
```
Comment (phildini) [inline README.md]: I think we don't know if it's None? Unless there's a source I don't know about.

File: README.md [hunk 2]
```diff
@@ -4,32 +4,32 @@ This is the running list of what in tech has been affected by COVID-19. Pull req
...
-| Slack | Encouraged, Required in JP | Restricted | ? | Restricted | 2020-03-04 |
-| [Square](https://twitter.com/zamosta/status/1234658276781912064) | Encouraged | Restricted | Restricted | Restricted | 2020-03-03 |
-| [Stripe](https://stripe.com/newsroom/news/covid-19) | Encouraged, some Required | Restricted | Restricted | ? |2020-02-29 |
-| [Textio](https://www.seattletimes.com/business/some-seattle-tech-companies-tell-employees-to-work-from-home-to-slow-spread-of-coronavirus/) | Encouraged | ? | ? | ? | 2020-03-04 |
-| [Twitter](https://blog.twitter.com/en_us/topics/company/2020/keeping-our-employees-and-partners-safe-during-coronavirus.html) | Required | Restricted | Restricted | Restricted | 2020-03-03 |
-| [Typless](https://typless.com/2020/03/05/switching-to-fully-remote-work/) | Required | Restricted | Restricted | Restricted | 2020-03-04 |
-| Yelp | Encouraged | Restricted | ? | ? | 2020-03-03 |
+| Company    | WFH Policy                                  | Travel     | Visitors   | Events     | Updated    | Source                                                                                                                                                | Po  |
```
Comment (phildini) [inline README.md]: What's the `Po` column for?

File: README.md (hunk 1 above)
Comment (trafnar) [reply README.md]: Oh, I should have mentioned this. It's based on my own specific knowledge talking to Apple employees. I have no source or way to verify that.

File: README.md (hunk 2 above)
Comment (trafnar) [reply README.md]: That was a mistake, fixed.

File: README.md (hunk 1 above)
Comment (trafnar) [reply README.md]: I've removed it.

**Comment by trafnar** [comment]
I updated a couple things you mentioned, working on getting that reflected in this PR

File: README.md [hunk 3]
```diff
@@ -4,32 +4,32 @@ This is the running list of what in tech has been affected by COVID-19. Pull req
...
-| [Textio](https://www.seattletimes.com/business/some-seattle-tech-companies-tell-employees-to-work-from-home-to-slow-spread-of-coronavirus/) | Encouraged | ? | ? | ? | 2020-03-04 |
-| [Twitter](https://blog.twitter.com/en_us/topics/company/2020/keeping-our-employees-and-partners-safe-during-coronavirus.html) | Required | Restricted | Restricted | Restricted | 2020-03-03 |
-| [Typless](https://typless.com/2020/03/05/switching-to-fully-remote-work/) | Required | Restricted | Restricted | Restricted | 2020-03-04 |
-| Yelp | Encouraged | Restricted | ? | ? | 2020-03-03 |
+| Company    | WFH Policy                                  | Travel     | Visitors   | Events     | Updated    | Source                                                                                                                                                |
+| ---------- | ------------------------------------------- | ---------- | ---------- | ---------- | ---------- | ----------------------------------------------------------------------------------------------------------------------------------------------------- |
```
Comment (phildini) [inline README.md]: I would prefer to keep these short because the width will always change based on added entries.

**Comment by trafnar** [comment]
this can be closed if #65 is accepted

**Comment by trafnar** [comment]
closing this in favor of #65

**Pull request closed without merging**